### ISSUE TITLE
Cannot assign reporter when creating a ticket

### ISSUE THREAD

**Issue by EventusTodd** [ankitpokhrel/jira-cli#521]
when doing a "jira issue create", you cannot use the option: -r "email"

**Please provide following details**
1. JiraCLI Version: 
    ```
    (Version="v1.1.0", GitCommit="a88eb61e142a6e604dc1f9f30884fff618277d22", CommitDate="2022-08-14T08:43:18+00:00", GoVersion="go1.17.11", Compiler="gc", Platform="linux/amd64")
    ```
2. Are you using Jira cloud or on-premise jira server? Also mention the version for on-premise installation.
    ```
    cloud
    ```
3. What operating system are you using? Also mention version.
    ```
    docker
    ```
4. What terminal are you using? Also mention version.
    ```
   macOS terminal
    ```

**To Reproduce**

1. jira issue create -r "email"

**Expected behavior**
I would like to assign a reporter thats not the default user thats logged in.

**Additional context**
I am using a bot user in jira that is creating the tickets. We have multiple users using the command line tool, but use the same linux user. For that case, I created a bot user that creats all the tickets. We would like to use the "-r" flag to change the reporter instead of going into jira manually.

**Comment by ankitpokhrel** [comment]
Duplicate of: https://github.com/ankitpokhrel/jira-cli/discussions/189

**Comment by ankitpokhrel** [comment]
Hi @EventusTodd, this is done with #539 and will be available in the next release. If you wish to use it for now, you can do so by installing the main branch using one of the following methods.

```sh
# Docker
$ docker run -it --rm ghcr.io/ankitpokhrel/jira-cli:main

# Golang
$ go install github.com/ankitpokhrel/jira-cli/cmd/jira@main

# Homebrew
$ brew tap ankitpokhrel/jira-cli
$ brew install jira-cli --HEAD
```

You can also compile the code locally using the steps [mentioned here](https://github.com/ankitpokhrel/jira-cli#development).

**Comment by ankitpokhrel** [comment]
This feature is available since [v1.3.0](https://github.com/ankitpokhrel/jira-cli/releases/tag/v1.3.0).
